### PR TITLE
Dragged note highlighting

### LINKED ISSUE #1885: Easier to move notes to different folder
Although the feature to move notes across folders exists, it's very finicky.

Ideally when moving notes to another folder I believe having the target folder be highlight when dragging over it. As of now there is no visual indication of where the note is going.

### PR DESCRIPTION
**Pull request by GuilhermeJSilva**
## Description
Changes the highlighting to more noticeable when  a note is dragged over a folder.

## Issue fixed
#1885 

## Type of changes

- :white_circle: Bug fix (Change that fixed an issue)
- :white_circle: Breaking change (Change that can cause existing functionality to change)
- :radio_button:  Improvement (Change that improves the code. Maybe performance or development improvement)
- :white_circle: Feature (Change that adds new functionality)
- :white_circle: Documentation change (Change that modifies documentation. Maybe typo fixes)

## Checklist:

- :radio_button: My code follows [the project code style](docs/code_style.md)
- :white_circle: I have written test for my code and it has been tested
- :radio_button: All existing tests have been passed
- :white_circle: I have attached a screenshot/video to visualize my change if possible

### CODE REVIEW

File: browser/main/SideNav/StorageItem.js [hunk 1]
```diff
@@ -291,16 +299,25 @@ class StorageItem extends React.Component {
...
           handleButtonClick={(e) => this.handleFolderButtonClick(folder.key)(e)}
           handleContextMenu={(e) => this.handleFolderButtonContextMenu(e, folder)}
           folderName={folder.name}
           folderColor={folder.color}
           isFolded={isFolded}
           noteCount={noteCount}
-          handleDrop={(e) => this.handleDrop(e, storage, folder, dispatch, location)}
-          handleDragEnter={this.handleDragEnter}
-          handleDragLeave={this.handleDragLeave}
+          handleDrop={(e) => {
```
Comment (ZeroX-DG) [inline browser/main/SideNav/StorageItem.js]: Can you move `e.prevenDefault()` to the `this.handleDrop`?

File: browser/main/SideNav/StorageItem.js [hunk 2]
```diff
@@ -291,16 +299,25 @@ class StorageItem extends React.Component {
...
+            e.preventDefault()
+            this.handleDrop(e, storage, folder, dispatch, location)
+          }}
+          handleDragEnter={(e) => {
```
Comment (ZeroX-DG) [inline browser/main/SideNav/StorageItem.js]: Can you move `e.prevenDefault()` to the `this.handleDrop`?

File: browser/main/SideNav/StorageItem.js [hunk 3]
```diff
@@ -291,16 +299,25 @@ class StorageItem extends React.Component {
...
+            e.preventDefault()
+            this.handleDragEnter(e, folder.key)
+          }}
+          handleDragLeave={(e) => {
+            e.preventDefault()
```
Comment (ZeroX-DG) [inline browser/main/SideNav/StorageItem.js]: Can you move `e.prevenDefault()` to the `this.handleDrop`?

**Comment by ehhc** [comment]
Thanks for implementing this. Never had time to do it myself but it is really needed (imho). Thanks :)

**Comment by GuilhermeJSilva** [comment]
Already changed the e.preventDefault()  to the respective handlers. @ehhc Your welcome, I'm glad to be part of the project.

**Pull request merged**